docs/api-testing/vault.md

```diff
@@ -43,16 +43,28 @@ Additionally, you can import variables from Postman. See [Importing Postman Coll
 If the same variable with the same name definition exists in both the **Project Vault** and **Company Vault**, then the value in the **Project Vault** will override the one in the **Company Vault**.
 :::
 
+### Mark Variables as Sensitive
+
+You can mark a variable as sensitive if it contains a sensitive value that you do not want to appear in your tests and test reports. When you mark a variable as sensitive, the value is obfuscated anywhere the value appears: in Vault, test, tests reports, and the project dashboard's logs and metrics when the variable is used in the URL, path, or query. 
+
+<img src={useBaseUrl('img/api-testing/sensitive-data.png')} alt="data obfuscated in report"/>
```

Comment:
This screenshot is blurry as well - it should be retaken with higher resolution. 
![Screenshot 2023-01-23 at 12 25 39 PM](https://user-images.githubusercontent.com/15352964/214108100-0e2af238-d035-4ec9-adc8-c2ba91cd372d.png)